### PR TITLE
Nav on scroll up

### PR DESCRIPTION
**Pull request by eliachoc**
Implementation of nav appearing on scroll up with colour change.

Some long pages like the 'service' and 'work' page can benefit from having a nav appearing to save users scrolling all the way to the top.

Be good to hear some of your UX feedback.

### CODE REVIEW

File: _layouts/about.html [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <!-- /.photo-container -->
 
     <section class="ppl-container">
-      <div class="ppl-inner grid">
+      <div class="ppl-inner grid" id="aboutDirectors">
```
Comment (shapeshed) [inline _layouts/about.html:27]: think this might be from another pull request but would favour the covention of `about-directors` on ids

**Comment by shapeshed** [comment]
:+1: I like this enhancement

**Comment by ggordan** [comment]
screenshot 2016-03-02 17 37 37

When the menu is expanded in the mobile view and you scroll up, the top link is partially hidden. Do we want to collapse the menu when the user scrolls up?

**Comment by petertait** [comment]
I would go with tweaking the styles so the menu is always visible, as opposed to watching for an upward scroll and hiding the menu.
